docs/scripting-ref.rst

```diff
@@ -2102,3 +2102,141 @@ Example:
              error("Splash 1.8 or newer required")
          end
      end
+
+.. _splash-mouse-click:
+
+splash:mouse_click
+------------------
+
+Trigger mouse click event in web page.
+
+**Signature:** ``splash:mouse_click(x, y)``
+
+**Parameters:**
+
+* x - integer or float with x position of element to be clicked
+* y - integer or float with y position of element to be clicked
+
+**Returns:** nil
+
+**Async:**: no.
+
+
+.. note::
+
+    Coordinates for mouse events must be relative to web document. Element on which action is performed must be inside viewport (must be visible
```

Comment:
in JavaScript you have browser window and document. For me this evaluates to

```javascript
window.innerWidth: 1855
window.innerHeight: 925
```

and remains constant no matter what you do (it doesn't change when you scroll). Window is just frame through which you view document. 

Document is what is inside web page, you can get its size by doing:
```javascript
> $(document).height()
9582
> $(document).width()
1840
```
 I was always thinking that by "viewport" we mean browser window, but it seems like this is not obvious and clear. Is my assumption correct? Coordinates of click should be coordinates inside web page, so they should be inside $(document), e.g. for this github page ATM you can click on elements with maximum y 9000.

Is this clear or am i missing something here?